### PR TITLE
Initial implementation of prune command

### LINKED ISSUE #92: Make a command to remove reputation
On our server we have backfilled reputation, but now there are old/inactive users in the top list and we would like to only have current users on there, so a command that removes/sets the reputation of a given user to a given number would be useful. Thank you!

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Add a command to remove reputation of a user.
Add a command to remove reputation of no longer present users

Close #92

### CODE REVIEW

File: src/main/java/de/chojo/repbot/commands/Prune.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            if (user.isEmpty()) return false;
+            if (Verifier.isValidId(user.get())) {
+                gdprService.cleanupGuildUser(eventWrapper.getGuild(), Long.valueOf(user.get()));
+                eventWrapper.reply(localizer.localize("command.prune.sub.user.removed")).queue();
+                return true;
+            }
+            var optUser = DiscordResolver.getUser(eventWrapper.getJda().getShardManager(), user.get());
+            if (optUser.isPresent()) {
+                gdprService.cleanupGuildUser(eventWrapper.getGuild(), optUser.get().getIdLong());
+                return true;
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/Prune.java:51]: Looks like the reply is missing here

File: src/main/resources/locale_de_DE.properties [hunk 2]
```diff
@@ -50,6 +50,12 @@ command.prefix.description=Zeigt den Präfix an.
 command.prefix.show=Das aktuelle Präfix ist %PREFIX%.
 command.prefix.sub.reset=Setzt den Präfix um
 command.prefix.sub.set=Setzt den Präfix
+command.prune.description=Entferne nicht mehr existierende oder einen bestimmten Nutzer.
+command.prune.sub.guild=Entferne Reputation von Nutzern, die nicht mehr auf dem Server sind.
+command.prune.sub.guild.done=%AMOUNT% Nutzer wurden entfernt.
+command.prune.sub.guild.started=Beginne mit entfernen von nicht existierenden Nutzern.
```
Comment (Taucher2003) [inline src/main/resources/locale_de_DE.properties:56]:
```suggestion
command.prune.sub.guild.started=Beginne mit dem Entfernen von nicht existierenden Nutzern.
```

File: src/main/java/de/chojo/repbot/commands/Prune.java (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/commands/Prune.java]: Indeed

**Review by Taucher2003 — APPROVED** [review]
Looks good to me

File: src/main/resources/locale.properties [hunk 3]
```diff
@@ -50,6 +50,12 @@ command.prefix.description=Show prefix
 command.prefix.show=The current prefix is %PREFIX%.
 command.prefix.sub.reset=Reset the prefix
 command.prefix.sub.set=Set the prefix
+command.prune.description=Prune invalid users or a specific user.
+command.prune.sub.guild=Remove reptutation of users which are no longer on the guild.
+command.prune.sub.guild.done=Pruned %AMOUNT% Users.
+command.prune.sub.guild.started=Starting removal of users which are no longer on the guild.
+command.prune.sub.user=Remove reputation of a user from the server.
```
Comment (yannicklamprecht) [inline src/main/resources/locale.properties:57]:
```suggestion
command.prune.sub.user=Remove reputation of an user from the server.
```

File: src/main/resources/locale_en_US.properties [hunk 4]
```diff
@@ -50,6 +50,12 @@ command.prefix.description=Show prefix
 command.prefix.show=The current prefix is %PREFIX%.
 command.prefix.sub.reset=Reset the prefix
 command.prefix.sub.set=Set the prefix
+command.prune.description=Prune invalid users or a specific user.
+command.prune.sub.guild=Remove reptutation of users which are no longer on the guild.
+command.prune.sub.guild.done=Pruned %AMOUNT% Users.
+command.prune.sub.guild.started=Starting removal of users which are no longer on the guild.
+command.prune.sub.user=Remove reputation of a user from the server.
```
Comment (yannicklamprecht) [inline src/main/resources/locale_en_US.properties:57]:
```suggestion
command.prune.sub.user=Remove reputation of an user from the server.
```

**Review by yannicklamprecht — CHANGES_REQUESTED** [review]

**Review by yannicklamprecht — APPROVED** [review]
Lgn

**Pull request merged**